### ISSUE TITLE
Alt Text property on images should be a text area

### ISSUE THREAD

**Issue by taylornowotny** [webstudio-is/webstudio#2383]
Per the design, all props with data type of string should be a text area.

In this case it would be nice if this was 3 lines by default. Not critical, but nice to have.

It's fine if "width" and "height" remain single-line, but only those.

This is relevant to the upcoming AI release becuase the AI will be able to generate alt text and we might use this layout in the AI tab.

<img width="688" alt="Screenshot 2023-09-26 at 1 11 21 PM" src="https://github.com/webstudio-is/webstudio/assets/86495896/c2b4a3b6-c991-4776-a142-c3c943e033b7">

https://www.figma.com/file/xCBegXEWxROLqA1Y31z2Xo/%F0%9F%93%96-Webstudio-Design-Docs?type=design&node-id=6107%3A55336&mode=design&t=E5z0cDBoEBuZy5jm-1

**Comment by taylornowotny** [comment]
closing becuase its covered by another issue
